docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md

```diff
@@ -172,7 +172,7 @@ Using the  <TechnicalTag relative="../../../" tag="data_context" text="Data Cont
 
 In the example, we have created a Datasource named `my_bigquery_datasource`, using the `add_or_update_sql` method and passing in a connection string.
 
-For more details on how to configure the BigQuery Datasource, please refer to [How to connect to a BigQuery database](../guides/connecting_to_your_data/database/bigquery.md)
+For more details on how to configure the BigQuery Datasource, please refer to [How to connect to a BigQuery database](/docs/0.15.50/guides/connecting_to_your_data/database/bigquery)
```

Comment:
```suggestion
To configure the BigQuery Datasource, see [How to connect to a BigQuery database](/docs/0.15.50/guides/connecting_to_your_data/database/bigquery).
```
![image](https://github.com/great-expectations/great_expectations/assets/104205257/e05456ea-7737-4bab-a3ee-241b5d140610)

Use _see_ instead of _refer to_.  See [Link text](https://developers.google.com/style/link-text).